### PR TITLE
Sticky form header for work tabs

### PR DESCRIPTION
**Pull request by katdivyareddy10**
Screen Shot 2020-05-21 at 2 18 54 PM
Screen Shot 2020-05-21 at 2 19 12 PM

Screen Shot 2020-05-21 at 2 45 09 PM

### CODE REVIEW

**Comment by katdivyareddy10** [comment]
@adamjarling I'm rebasing from deploy/staging after the merges went through for other PR's

**Comment by adamjarling** [comment]
@katdivyareddy10 Mind rebasing this off `deploy/staging` and re-pushing?

**Comment by adamjarling** [comment]
The functionality is awesome!  Wow after being in that super-long form all week, this is like heaven for the UX.  Nice work.

File: assets/js/components/Work/Tabs/Header.jsx [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+      zIndex: 1110,
+      padding: "0 1rem",
+    },
+  };
+
+  useEffect(() => {
+    const headerElement = document.getElementById("main-navigation")
+      ? document.getElementById("main-navigation").getBoundingClientRect()
+      : {};
+    setHeaderHeight(headerElement.height);
```
Comment (adamjarling) [inline assets/js/components/Work/Tabs/Header.jsx:20]: No big deal, but you could also write this as:
```
setHeaderHeight(document.getElementById("main-navigation")
      ? document.getElementById("main-navigation").getBoundingClientRect()
      : 0
);
```

As is, it'll set header height to `undefined` if the `headerElement` doesn't exist for some reason.

**Pull request merged**